src/endpoints/applecrashreport.rs

```diff
@@ -156,3 +156,69 @@ pub fn configure(app: ServiceApp) -> ServiceApp {
             .with(handle_apple_crash_report_request);
     })
 }
+
+#[cfg(test)]
+mod tests {
+    use std::fs;
+
+    use actix_web::test::TestServer;
+    use reqwest::{multipart, Client, StatusCode};
+
+    use crate::app::ServiceState;
+    use crate::config::Config;
+    use crate::test;
+    use crate::types::SymbolicationResponse;
+
+    #[tokio::test]
+    async fn test_basic() {
+        test::setup();
+
+        let service = ServiceState::create(Config::default()).unwrap();
+        let server = TestServer::with_factory(move || crate::server::create_app(service.clone()));
+
+        let file_contents = fs::read("tests/fixtures/apple_crash_report.txt").unwrap();
```

Comment:
Is this really safe?  AFAIK you could be running the test binary from any cwd so the only safe thing to do is `include_bytes`()`.  But I could be wrong.

Reply:
All of our tests assume that you run the test binary via `cargo test`, and that guarantees a stable CWD.